graylog2-web-interface/test/helpers/suppressConsole.ts

```diff
@@ -23,14 +23,17 @@
  *
  * @param {function} fn - the function which should be called after disabling `console.error` and before restoring it.
  */
-const suppressConsole = (fn: () => void) => {
+const suppressConsole = <T extends () => any>(fn: T): Promise<ReturnType<T>> => {
```

Comment:
A slightly simpler way to type this would be:
```suggestion
const suppressConsole = <R>(fn: () => R): Promise<R> => {
```